### DISCUSSION TITLE
Welcome to pandas-bootstrap Discussions!

### DISCUSSION THREAD

**Discussion by williambdean** [williambdean/pandas-bootstrap#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by iamasy8** [comment]
Hi idk yet how this thing really works lol im not giving it much of time ig ! but today i just started bootstrap and i searched the name than this open dics appeared first . so i hope that i can meet new passionate  ppl who i can build projects with and exchange information :)

**Reply by williambdean** [reply]
Sounds great. Well if you have any specific questions about the documentation or the code, feel free to either raise an issue or start another discussion
